app/components/toggle.hbs

```diff
@@ -35,4 +40,8 @@
       </svg>
     </span>
   </span>
+
+  {{#if (and @tooltipCopy (not-eq @tooltipCopy ""))}}
+    <EmberTooltip @tooltipClass="normal-case font-normal" @text={{@tooltipCopy}} />
```

Comment:
@libmartinito this isn't a very "flexible" interface - every time we have to pass an option to the `EmberTooltip` component (you might want to configure the `@position` for example), you have to add it to the `Toggle` component's interface. 

The standard way to do this is to instead wrap this in the caller. Example: 

```hbs
<div class="flex">
  <Toggle />
  <EmberTooltip />
</div>
```

This gives the caller full flexibility on what they pass to `EmberTooltip` - it also gives them flexibility on whether they want to show a tooltip, popover etc.